Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb

```diff
@@ -0,0 +1,1438 @@
+{
```

Comment:
<p>Could you move</p><blockquote>To understand model performance, dividing the dataset into a training set and a test set is a good strategy. </blockquote><p>to another line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Done